backend/LexBoxApi/Services/HgService.cs

```diff
@@ -271,8 +271,19 @@ private string[] GetWsList(System.Xml.XmlElement root, string tagName)
     {
         var result = await ExecuteHgCommandServerCommand(code, "flexmodelversion", token);
         var text = await result.ReadAsStringAsync(token);
-        var json = JsonDocument.Parse(text);
-        return json.RootElement.GetProperty("modelversion").GetInt32();
+        if (string.IsNullOrEmpty(text)) return null;
+        try
+        {
+            var json = JsonDocument.Parse(text);
+            if (json.RootElement.GetProperty("modelversion").TryGetInt32(out int version)) return version;
```

Comment:
`GetProperty` will throw a `KeyNotFoundException` if it's not found, it looks like there's a try get property method as well

Reply:
Handled in 075cd630.